spec/requests/works_spec.rb

```diff
@@ -313,6 +314,11 @@
           )
         end
 
+        let(:contributors) do
+          { '999' =>
+            { '_destroy' => 'false', 'full_name' => 'Stanford', 'role_term' => 'organization|Host institution' } }
```

Comment:
Don't all three name attrs come across in both cases? might want to add first name and last name here (set to empty strings) for higher fidelity w/ the deployed app.

Reply:
Yup. Done.